standard/variables.md

```diff
@@ -1012,50 +1012,50 @@ A *reference variable* is a variable that refers to another variable, called the
 >
 > *end example*
 
-A `ref struct` may include `ref struct` or ref-like fields. A *ref-like field* refers to the same storage as its initializing variable. Its storage size in the ref struct is the same storage size of a reference field. Unlike a reference field, a ref-like field may refer to a `struct` whose storage may be on the execution stack. Ref struct types include `Span<T>`, `ReadOnlySpan<T>`, and other types that include an unmanaged pointer as a member.
+A *reference return* is the expression returned by reference from a method whose return type includes the `ref` or `ref readonly` modifiers (§15.6.1). The variable expression of the *reference return* is the *referent* of the *reference return*.
 
-> *Example:* The following example demonstrates a `ReadOnlySpan<T>` *reference variable* whose *referent* is the characters in a string. `ReadOnlySpan<T>` is a `ref struct`. One of its internal fields is a *ref-like field* that refers to the character array in the `string` referent. Therefore, `chars` is a *reference variable*:
+> *Example:* The following example demonstrates a =*reference return* whose *referent* is an element of an array field:
 >
 > ```csharp
-> string Reverse(string s)
+> public class C
 > {
->     ReadOnlySpan<char> chars = s.AsSpan();
->     var reversed = new char[chars.Length];
->     for (int i = 0; i < chars.Length; i++)
+>     private int[] arr = new int[10];
+>
+>     public readonly ref int M()
 >     {
->         reversed[chars.Length-i-1] = chars[i];
->     }
->     return new string(reversed);
+>         // element is a reference variable that refers to arr[5]
+>         ref int element = ref arr[5];
+>         return ref element; // return reference to arr[5];
+>     }     
 > }
 > ```
 >
 > *end example*
 
-A *reference return* is the expression returned by reference from a method whose return type includes the `ref` or `ref readonly` modifiers (§14.6.1). The variable expression of the *reference return* is the *referent* of the *reference return*.
+A `ref struct` may include `ref struct` or ref-like fields. A *ref-like field* refers to the same storage as its initializing expression. The initializing expression of the *ref_like field* is its *referent*. Unlike a reference field, a ref-like field may refer to a `struct` whose storage may be on the execution stack. Ref struct types include `Span<T>`, `ReadOnlySpan<T>`, and other types that include an unmanaged pointer as a member.
```

Comment:
I moved this description below the `return ref` description.

The one semantic change here says that `ref struct` types cannot be copied by value beyond the *ref_safe_scope* of its initializing expressions.